From ec9a60dbc6d5eaf39fccdb50352deb990808bf00 Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Tue, 12 Dec 2017 05:43:22 -0600
Subject: [PATCH] "0001-linker-add-hwvendor-to-link-location
 _______________________________

*Previous From 00194a2f67cccea047dcb7e740d06bf6ef81430b Mon Sep 17 00:00:00 2001
*Previous From: Meticulus <theonejohnnyd@gmail.com>
*Previous Date: Thu, 16 Nov 2017 10:50:23 -0600
*Previous Subject: [PATCH] linker: add hwvendor to link location
*
*Change-Id: I9f29c6e22c613dec7f30e91be5e273548821654f"
---
 linker/linker.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/linker/linker.cpp b/linker/linker.cpp
index 3d6fdf9..f591bb9 100644
--- a/linker/linker.cpp
+++ b/linker/linker.cpp
@@ -85,11 +85,13 @@ static const char* const kLdConfigFilePath = "/system/etc/ld.config.txt";
 #if defined(__LP64__)
 static const char* const kSystemLibDir     = "/system/lib64";
 static const char* const kVendorLibDir     = "/vendor/lib64";
+static const char* const kHwVendorLibDir   = "/hwvendor/lib64";
 static const char* const kAsanSystemLibDir = "/data/asan/system/lib64";
 static const char* const kAsanVendorLibDir = "/data/asan/vendor/lib64";
 #else
 static const char* const kSystemLibDir     = "/system/lib";
 static const char* const kVendorLibDir     = "/vendor/lib";
+static const char* const kHwVendorLibDir   = "/hwvendor/lib";
 static const char* const kAsanSystemLibDir = "/data/asan/system/lib";
 static const char* const kAsanVendorLibDir = "/data/asan/vendor/lib";
 #endif
@@ -99,6 +101,7 @@ static const char* const kAsanLibDirPrefix = "/data/asan";
 static const char* const kDefaultLdPaths[] = {
   kSystemLibDir,
   kVendorLibDir,
+  kHwVendorLibDir,
   nullptr
 };
 
-- 
2.7.4

